cloud/passportauth/src/auth/DefaultStrategy.ts

```diff
@@ -1,31 +1,29 @@
 import { BunyanLogger, Logger } from '@raincatcher/logger';
-import { UserSecurityService } from '../user/UserSecurityService';
+import { UserRepository } from '../user/UserRepository';
+import { UserService } from '../user/UserService';
 
 const log: Logger = new BunyanLogger({ name: 'Passport-Auth', level: 'error' });
-import { User } from '../user/User';
 
 /**
  * Default strategy to be used by Passport's local strategy. If user credentials are valid, proceed to login,
- * otherwise, reject it. Calls serialize user with user login id.
+ * otherwise, reject it.
  *
- * @param userSec - security service
+ * @param userRepo - User repository
+ * @param userService - User service
  * @returns {Function} - Returns the default strategy function to be used by passport
  */
-export const defaultStrategy = (userSec: UserSecurityService) => {
+export const defaultStrategy = (userRepo: UserRepository, userService: UserService) => {
   return (loginId: string, password: string, done: (error: Error | null, user: any) => any) => {
-    userSec.getUserByLogin(loginId).then((user: User) => {
-      if (!user) {
-        return done(null, false);
+    userRepo.getUserByLogin(loginId).then((user: any) => {
+      if (user && userService.getPassword(user) === password) {
```

Comment:
Same as above. I think it''s cleaner and more flexible to just use password matcher method and pull out this code to example application and demo which will not use hashes.